src/main.cpp

```diff
@@ -3315,7 +3311,7 @@ bool TestBlockValidity(CValidationState &state, const CBlock& block, CBlockIndex
         return false;
     if (!ContextualCheckBlock(block, state, pindexPrev))
         return false;
-    if (!ConnectBlock(block, state, &indexDummy, viewNew, true))
+    if (!ConnectBlock(block, state, &indexDummy, viewNew, verifier, true))
```

Comment:
Need a regular verifier here, not the disabled one from above (which is passed into `CheckBlock`).